staging/src/k8s.io/apiextensions-apiserver/pkg/controller/status/naming_controller.go

```diff
@@ -76,7 +77,7 @@ func NewNamingConditionController(
 	}
 
 	informerIndexer := crdInformer.Informer().GetIndexer()
-	c.crdMutationCache = cache.NewIntegerResourceVersionMutationCache(informerIndexer, informerIndexer, 60*time.Second, false)
+	c.crdMutationCache = cache.NewIntegerResourceVersionMutationCache(ctx, informerIndexer, informerIndexer, 60*time.Second, false)
 
 	crdInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
I don't understand why some call sites were updated to pass the context to AddEventHandlerWithContext and some weren't